src/drt/src/dr/FlexDR.cpp

```diff
@@ -963,21 +926,8 @@ frCoord FlexDR::init_via2viaMinLenNew_minSpc(frLayerNum lNum,
   width1 = viaBox1.minDXDY();
   prl1 = isCurrDirX ? (viaBox1.yMax() - viaBox1.yMin())
                     : (viaBox1.xMax() - viaBox1.xMin());
-  reqDist = 0;
-  auto con = getTech()->getLayer(lNum)->getMinSpacing();
-  if (con) {
-    if (con->typeId() == frConstraintTypeEnum::frcSpacingConstraint) {
-      reqDist = static_cast<frSpacingConstraint*>(con)->getMinSpacing();
-    } else if (con->typeId()
-               == frConstraintTypeEnum::frcSpacingTablePrlConstraint) {
-      reqDist = static_cast<frSpacingTablePrlConstraint*>(con)->find(
-          max(width1, width2), prl1);
-    } else if (con->typeId()
-               == frConstraintTypeEnum::frcSpacingTableTwConstraint) {
-      reqDist = static_cast<frSpacingTableTwConstraint*>(con)->find(
-          width1, width2, prl1);
-    }
-  }
+  reqDist = getTech()->getLayer(lNum)->getMinSpacingValue(
+      width1, width2, min(prl1, prl2), false);
```

Comment:
min(prl1, prl2) looks to be a mistranslation - should be prl1

Reply:
Good catch, fixed